frontend/src/features/ratings/RatingsFilter.tsx

```diff
@@ -4,13 +4,24 @@ import { Box, Collapse, Grid } from '@mui/material';
 
 import { CollapseButton } from 'src/components';
 import { useListFilter } from 'src/hooks';
+
 import IsPublicFilter from './IsPublicFilter';
 import MarkAllRatingsMenu from './MarkAllRatings';
+import RatingOrderByInput from './RatingOrderByInput';
 
-function RatingsFilter() {
+function RatingsFilter({
+  defaultFilters = [],
+}: {
+  // A list of default values used to initialize the filters.
+  defaultFilters: Array<{ name: string; value: string }>;
+}) {
   const { t } = useTranslation();
   const [expanded, setExpanded] = useState(false);
-  const [filterParams, setFilter] = useListFilter();
+  const [filterParams, setFilter] = useListFilter({ defaults: defaultFilters });
+
+  const defaultFilterOrderBy = defaultFilters
+    .filter((param) => param.name === 'orderBy' && param.value != null)
+    .pop() ?? { value: '' };
```

Comment:
This is unnecessary, isn'it ? `filterParams.get('orderBy') ?? ""` can be used directly.

```suggestion
```

Reply:
Indeed, fixed by https://github.com/tournesol-app/tournesol/pull/1181/commits/3be9baf39d85d851ae5e95a0ebb80981be6ef2d9